proxy/cypress/integration/machines/list.spec.js

```diff
@@ -0,0 +1,31 @@
+import { customAlphabet } from 'nanoid'
+
+import { generateMac, login } from "../utils";
+
+const nanoid = customAlphabet('1234567890abcdefghi', 10);
```

Comment:
nanoid generates ids with `_` by default which are invalid for `machine.hostname`.